src/client/Microsoft.Identity.Client/AuthScheme/PoP/InMemoryCryptoProvider.cs

```diff
@@ -27,7 +28,14 @@ public InMemoryCryptoProvider()
 
         private void InitializeSigningKey()
         {
+#if NETFRAMEWORK
+            // This method was obsolete in .NET,
+            // but Create() on .NET FWK defaults to PKCS1 padding.
+            _signingKey = RSA.Create("RSAPSS");
+#else
             _signingKey = RSA.Create();
```

Comment:
What's the default for .NET Standard? Should we explicitly specify RSAPSS for all targets, not rely one default?

Reply:
The method `RSA.Create("RSAPSS")` is obsolete (returns NULL) on .NET and .NETSTANDARD. On these tfms, `RSA.Create()` factory returns a modern implementation of RSA, which uses PSS padding by default. 

On .NET FWK however, `RSA.Create()` returns a old implementation of RSA (RsaCryptoProvider) which uses PKCS1 padding by default. `RSA.Create("RSAPSS")` is NOT deprecated on .NET FWK.

I didn't find a way to change padding after the RSA object is created. The RSA object is read-only. So it has to be done via the ctor.